python/cudf/cudf/utils/ioutils.py

```diff
@@ -1155,6 +1161,10 @@ def get_filepath_or_buffer(
             if os.path.exists(paths[0]):
                 path_or_data = paths if len(paths) > 1 else paths[0]
 
+        elif _is_s3_filesystem(fs):
+            fs = pyarrow.fs.S3FileSystem()
+            path_or_data = fs.open_input_file(paths[0])
+
```

Comment:
Can remove this since we don't want to go down this codepath for anything that's not `read_csv` and this utility is called by other readers as well.